config/routes.rb

```diff
@@ -209,6 +209,10 @@
           resources :materials, path: 'files'
         end
       end
+
+      resources :course_users, only: [] do
+        resources :experience_points_records, only: [:new, :create, :index]
```

Comment:
OK.

Then change the `resources :course_users` to be a scope. There is no controller for the Course User resource.